### ISSUE TITLE
[BUG] Small but annoying spelling mistake in Russian localization

### ISSUE THREAD

**Issue by pontifex91** [SteamDeckHomebrew/decky-loader#519]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [common issues section in the readme file](https://github.com/SteamDeckHomebrew/decky-loader#-common-issues)
- [ ] I have attached logs to this bug report (failure to include logs will mean your issue may not be responded to).

### Bug Report Description

In the plugin store, when installing a plugin, a popup appears "Are you sure *?".
In Russian localization, a spelling mistake was made in the word "увере**нн**ы".
This mistake occurs in both the stable branch and v2.10.4-pre6

![20230730113104_1](https://github.com/SteamDeckHomebrew/decky-loader/assets/8258184/01cd6e8c-8968-4286-9355-6ce76bbe92f0)

### Expected Behaviour

The correct spelling should be:
"увере**н**ы"
That is, with one letter "н", and not with two.
It may seem insignificant, but in Russian the word with two "н" is used in a different context, for example, if the sentence sounded like  "Эти плагины стабильны и уверенны (This plugins are stable and sure)", then the use of two "н" would be justified. Source and detailed explanation of the difference: https://russkiiyazyk.ru/orfografiya/pravopisanie/uvereny-ili-uverenny-kak-pishetsya.html

To summarize:
Suggestion" Вы увере**нн**ы, что хотите установить *?"
Need to change to "Вы увере**н**ы, что хотите установить *?"

### SteamOS version

3.4.8

### Selected Update Channel

Prerelease

### Logs

-

**Comment by RodoMa92** [comment]
You can find the corresponding string here and make the change yourself https://weblate.werwolv.net/translate/decky/decky/ru/?checksum=a4acf9870d2730d6&sort_by=-priority,position

**Comment by pontifex91** [comment]
Thanks, I've made a few changes to the Weblate translation, should the ticket be closed here?

**Comment by RodoMa92** [comment]
Yup, closing this. It will be merged on the next pull from weblate.
